.eslintignore

```diff
@@ -1,5 +1,6 @@
 node_modules/*
 modules/*/js/*.js
+modules/*/js/components/*.js
```

Comment:
```suggestion
modules/*/js/*/*.js
```

I don't think there's any subdirectory of compiled js files that are supposed to be analyzed